molecule/debian/molecule.yml

```diff
@@ -0,0 +1,67 @@
+---
+# This molecule configuration file is suitable for testing Ansible
+# roles that _do_ require SystemD.  If your Ansible role _does not_
+# require SystemD then you should use molecule-no-systemd.yml instead.
+#
+# Note that the molecule configuration file that is symlinked to
+# molecule.yml is the one that will be used.
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint: |
+  set -e
+  yamllint .
+  ansible-lint
+  flake8
+platforms:
+  - name: debian10_systemd
+    image: geerlingguy/docker-debian10-ansible:latest
+    privileged: yes
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    command: /lib/systemd/systemd
+    pre_build_image: yes
+  - name: debian11_systemd
+    image: geerlingguy/docker-debian11-ansible:latest
+    privileged: yes
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    command: /lib/systemd/systemd
+    pre_build_image: yes
+provisioner:
+  name: ansible
+  inventory:
+    host_vars:
+      debian9_systemd:
```

Comment:
If we're not testing against Debian 9 (Stretch) is there any reason to keep all of this?

Reply:
The intent is to remove the Debian 9 jazz wholesale once CyHy moves on from the dark ages.  I can remove it from this particular repo.

Reply:
Please see commit 57517ab.